.eslintrc.js

```diff
@@ -19,16 +18,20 @@ module.exports = {
   rules: {
     '@typescript-eslint/interface-name-prefix': 'off',
     '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
-    'max-len': ['warn', {
-      code: 100,
-      tabWidth: 2,
-      ignoreComments: false,
-      ignoreTrailingComments: true,
-      ignoreUrls: true,
-      ignoreStrings: true,
-      ignoreTemplateLiterals: true,
-      ignoreRegExpLiterals: true
-    }]
+    'max-len': [
+      'warn',
+      {
+        code: 100,
+        tabWidth: 2,
+        ignoreComments: false,
+        ignoreTrailingComments: true,
+        ignoreUrls: true,
+        ignoreStrings: true,
+        ignoreTemplateLiterals: true,
+        ignoreRegExpLiterals: true,
+      },
+    ],
```

Comment:
Could you add `'comma-dangle': ['warn', 'always-multiline'],` ? Otherwise we have a conflict with eslint vs prettier. Eslint wants to remove the comma at the last multiline item and prettier automatially adds it. This makes it compatible.

Reply:
eslint _shouldn't_ complain because of the extension of prettier, but I'll add it just in case.